src/themes/greyVest/index.js

```diff
@@ -923,11 +923,11 @@ export let Pager = props => (
   />
 )
 
-export let PagedResultTable = ({ tree, node, ...props }) => (
+export let PagedResultTable = ({ tree, node, path, ...props }) => (
   <>
-    <ExampleTypes.ResultTable tree={tree} node={node} {...props} />
+    <ExampleTypes.ResultTable {...{ tree, node, path, ...props }} />
     <Flex style={{ justifyContent: 'space-around', padding: '10px' }}>
-      <Pager tree={tree} node={node} />
+      <Pager {...{ tree, node, path }} />
```

Comment:
This is not related to the rest of the PR, just a small fix for a bug that snuck in with #277 - now that we're not wrapping PagedResultTable in `contexturify`, we need to pass along `path` so that Pager can look up the node on its own, or else it fails and throws an error.